src/test/java/Homework18_PB.java

```diff
@@ -14,6 +20,36 @@ public class Homework18_PB extends BaseTest{
     12. Create a pull request.*/
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.